pkg/goldpinger/k8s.go

```diff
@@ -103,6 +103,17 @@ func getPodNodeName(p v1.Pod) string {
 	return p.Name
 }
 
+// isReady is equivalent to k8s.io/kubectl/pkg/util/podutils.IsPodReady" but that brings us up a go version
+// if we don't mind those dependencies could replace.
```

Comment:
going to latest dependencies brough it go 1.24 in go mod assume you'd want that as seperate change.